zokrates_core/src/proof_system/scheme/pghr13.rs

```diff
@@ -1,5 +1,5 @@
 use crate::proof_system::scheme::{NonUniversalScheme, Scheme};
-use crate::proof_system::solidity::{SOLIDITY_G2_ADDITION_LIB, SOLIDITY_PAIRING_LIB};
+use crate::proof_system::solidity::{solidity_pairing_lib, SOLIDITY_G2_ADDITION_LIB};
```

Comment:
I guess `SOLIDITY_G2_ADDITION_LIB` should be removed here, and the usage below in the `format!`
https://github.com/Zokrates/ZoKrates/blob/38f0b4a83a673de390b9da0cdfd21d74b8337fa7/zokrates_core/src/proof_system/scheme/pghr13.rs#L141